app/workers/sync_courses_from_find.rb

```diff
@@ -0,0 +1,182 @@
+class SyncCoursesFromFind
+  attr_reader :provider
+
+  include Sidekiq::Worker
+  sidekiq_options retry: 3, queue: :low_priority
+
+  def perform(provider_id, provider_recruitment_cycle_year)
+    @provider = Provider.find(provider_id)
+    @provider_recruitment_cycle_year = provider_recruitment_cycle_year
+
+    find_provider.courses.each do |find_course|
+      create_or_update_course(find_course)
+    end
+  end
+
+private
+
+  def find_provider
+    # https://api2.publish-teacher-training-courses.service.gov.uk/api/v3/recruitment_cycles/2020/providers/1N1/?include=sites,courses.sites
```

Comment:
I think the comment is out of date, it's probably this now, but could you sense check it?

```suggestion
    # https://api2.publish-teacher-training-courses.service.gov.uk/api/v3/recruitment_cycles/2020/providers/1N1/?include=sites,courses.subjects,courses.sites,courses.site_statuses.site
```